jira_agile_metrics/calculators/cycletime.py

```diff
@@ -43,7 +43,7 @@ def run(self, now=None):
             self.query_manager,
             self.settings['cycle'],
             self.settings['attributes'],
-            self.settings['backlog_column'],
+            self.settings['committed_column'],
```

Comment:
Can you explain why we are changing this? Does it alter the meaning of the configuration options? I'm very worried about breaking backwards compatibility with existing configurations in the wild.

Reply:
As described in https://github.com/DeloitteDigitalUK/jira-agile-metrics/pull/27#issuecomment-636308642 I think that cycle time calculations require `Committed column` and `Done column` only.